src/components/Search/Search.tsx

```diff
@@ -10,10 +10,22 @@ import Meta from '../Meta'
 import { canSearchForProperty } from '@/utils/userPermissions'
 import { PropertyListItem } from '@/models/propertyListItem'
 
-const Search = ({ query }) => {
+interface SearchProps {
+  query?: {
+    searchText?: string
+    pageNumber?: string
+  }
+}
+
+interface PropertyData {
+  total: string
```

Comment:
Based on the line 90, I'm guessing it actually comes through as string from the API?
```js
setSearchHitTotal(parseInt(propertiesData.total))
```

Reply:
Hmm the API seems to be returning a `long` [[link to model](https://github.com/LBHackney-IT/repairs-api-dotnet/blob/423055144f1f7df6a8942c94f3d2c0906c0bca9b/RepairsApi/V2/Boundary/Response/SearchForPropertiesResponse.cs#L10)].

Reply:
its a number